drivers/espi/espi_mchp_xec_host_v2.c

```diff
@@ -523,7 +534,117 @@ static int init_acpi_ec1(const struct device *dev)
 #undef	INIT_ACPI_EC1
 #define	INIT_ACPI_EC1		init_acpi_ec1
 
-#endif /* CONFIG_ESPI_PERIPHERAL_HOST_IO_PVT */
+#endif /* CONFIG_ESPI_PERIPHERAL_EC_HOST_CMD || CONFIG_ESPI_PERIPHERAL_HOST_IO_PVT */
+
+#ifdef CONFIG_ESPI_PERIPHERAL_EC_HOST_CMD
+
+BUILD_ASSERT(DT_NODE_HAS_STATUS(DT_NODELABEL(emi0), okay),
+	     "XEC EMI0 DT node is disabled!");
+
+struct xec_emi_config {
```

Comment:
Perhaps move this definition in header or together with other driver private structures

Reply:
Followed same approach as in CONFIG_ESPI_PERIPHERAL_XEC_MAILBOX (line 100) and CONFIG_ESPI_PERIPHERAL_8042_KBC (line 163).